package.json

```diff
@@ -191,24 +190,22 @@
     "@types/parse-gitignore": "1.0.2",
     "@types/prettyjson": "0.0.33",
     "@types/semver": "7.5.8",
-    "@types/uuid": "9.0.8",
     "@types/ws": "8.5.14",
     "@vitest/coverage-v8": "1.6.0",
-    "c8": "9.1.0",
+    "c8": "10.1.3",
     "cheerio": "1.0.0",
     "eslint-plugin-sort-destructure-keys": "2.0.0",
     "eslint-plugin-workspace": "file:./tools/lint-rules",
     "form-data": "4.0.1",
-    "fs-extra": "11.2.0",
```

Comment:
unrelated but this was easy to get rid of so I did (just a dev dep, so won't help with package weight, but still)

Reply:
pulled out into https://github.com/netlify/cli/pull/7025/